testdata/local/admin.json

```diff
@@ -1,11 +1,38 @@
-{ "_id" : "ui", "default_project" : "evergreen", "url" : "http://localhost:9090", "http_listen_addr" : ":9090", "secret" : "this is a secret", "cors_origins": ["http://localhost:3000","http://localhost:5173","http://localhost:4173"], "file_streaming_content_types": ["application/json", "text/plain", "text/csv"], "userVoice": "https://uservoice.com", "uiv2_url": "http://localhost:3000", "parsley_url": "http://localhost:4173"}
```

Comment:
Is al the other local admin data supposed to be deleted? The new file doesn't look like it's the local admin settings anymore.

Reply:
Relic of my own testing 😟 removed.